### DISCUSSION TITLE
20240224-design-system/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#53] [Comments]
# 20240224-design-system/

인프랩이 지난 2년간 플랫폼 팀 없이 오픈 소스를 이용해 디자인 시스템을 만들고 운영한 경험과 그 과정에서 했던 고민들을 공유합니다.

https://tech.inflab.com/20240224-design-system/

**Comment by hyejin30** [comment]
자세한 정보를 알려주셔서 감사합니다 최고예요!!

**Comment by Har-ii** [comment]
최고예요

**Comment by buddle6091** [comment]
사내에서 혼자 프론트를 맡아 개발하는데, 좋은 사수가 된 글이었습니다. 감사합니다 :)

**Comment by developer-choi** [comment]
혹시 radix-ui처럼 Headless UI는 선택지에 없었나요? 히스토리가 궁금해요!
